mood.txt

```diff
@@ -0,0 +1,2 @@
+{"0": "lopā"}
```

Comment:
Piedāvāju šo failu izdzēst un pielikt .gitignore, ka šo mums nevajag kodā.

Un attiecīgi tur, kur saglabājam dotajā failā pirms saglabāšanas, ja fails neeksistē, tad to izveidot. Man liekas, ka open funkcija, kas tiek izmantota to nodrošina, ja nē, tad jāpadod kā otrs parametrs cits karogs. Jāpamēģina.